app/test/priority_test.go

```diff
@@ -80,6 +80,7 @@ func (s *PriorityTestSuite) TestPriorityByGasPrice() {
 	wg := &sync.WaitGroup{}
 	for _, signer := range s.signers {
 		wg.Add(1)
+		signer := signer // new variable per iteration
```

Comment:
while this gets rid of the linting error, does this actually get rid of the problem?

would it be best to pass this to the goroutine?

Reply:
The problem doesn't actually exist for us because we're on Go 1.22. Go 1.22 has new lifetime rules:

>  In Go 1.22, loop variables are created anew for each iteration, so such references are no longer at risk of using a variable after it has been updated by the loop.

Ref: https://tip.golang.org/doc/go1.22

My hunch is that `go vet` in my editor (potentially others) is using an older version of Go.

Reply:
Woah, dope